driver/conv_driver.hpp

```diff
@@ -1997,6 +2017,8 @@ int ConvDriver<Tgpu, Tref>::RunForwardCPU()
 template <typename Tgpu, typename Tref>
 int ConvDriver<Tgpu, Tref>::RunForwardGPUReference()
 {
+    AutoConvDirectNaiveAlwaysEnable auto_naive_direct_naive_always_enable;
```

Comment:
[Recommendation] Use shorter name (unless you are thinking this code is ugly and so you would like to emphasize it). Everywhere.

Reply:
OK, changed to a shorter name `naive_conv_enable`